src/olympia/editors/templates/editors/review.html

```diff
@@ -116,6 +116,26 @@ <h3 id="history">
               {% set record = record_version.activity_log %}
               {% include 'editors/includes/history.html' %}
             {% endfor %}
+            {% if is_post_reviewer and version.is_ready_for_auto_approval %}
+              {# If it was not auto-approved, but could have been (i.e. it's a listed, unreviewed webextension update of a public extension) try to find out why. #}
+              <tr>
+                <th>{{ _('Auto Approval Summary') }}</th>
+                <td class="auto_approval">
+                  {% if version.autoapprovalsummary %}
+                    {# Call calculate_verdict() again, it will use the data already stored. Need to pass max_average_daily_users and min_approved_updates current values. #}
+                    {% set verdict_info=version.autoapprovalsummary.calculate_verdict(max_average_daily_users=auto_approval['max_average_daily_users'], min_approved_updates=auto_approval['min_approved_updates']) %}
+                    {% set verdict_info_human=version.autoapprovalsummary.verdict_info_prettifier(verdict_info) %}
```

Comment:
I was afraid you'd say that, because I know you're right :)

Main issue I had with that approach is that the versions queryset must be kept lazy, I didn't want to introduce a new transformer (I'd rather get rid of our transformers...) and `prefetch_related` is insufficient for this use-case.

I'll try harder, it looks like I could use the queryset after it has been paginated.